rtt_roscomm/include/rtt_roscomm/passthrough_callback_queue.hpp

```diff
@@ -0,0 +1,232 @@
+
+#ifndef RTT_ROSCOMM__PASSTHROUGH_CALLBACK_QUEUE_HPP_
+#define RTT_ROSCOMM__PASSTHROUGH_CALLBACK_QUEUE_HPP_
+
+#include <ros/ros.h>
+#include <ros/callback_queue_interface.h>
+#include <ros/subscription_queue.h>
+#include <ros/callback_queue.h>
+
+namespace rtt_roscomm {
+  class PassthroughCallbackQueue: public ros::CallbackQueueInterface
+  {
+    public:
+      PassthroughCallbackQueue();
+
+      /** 
+       * Implementation of ros::CallbackQueueInterface::addCallback()
+       * 
+       * @param callback aaa
+       * @param owner_id aaa
+       * 
+       * @return void
+       */
```

Comment:
Either you should explain what this method is doing internally (i.e. invoke the callback directly), or omit the code documentation. Like this, it is pretty useless.

Same for `removeByID()`, which is a no-op. Could be implemented in the header to allow the compiler to inline it, and to document without the need of an additional comment that it is not doing anything here.